app/lib/model-helpers.js

```diff
@@ -3,15 +3,15 @@ const extractEmail = (contacts) => {
   if (!contacts || contacts.length === 0) {
     return ''
   }
-  const email = contacts.filter(x => x.contact.contact_type_i === 2).sort(propertyComparatorDesc('contact', 'id')).map(y => y.contact.contact)
+  const email = contacts.filter(x => x.contact.contact_type_id === 2).sort(propertyComparatorDesc('contact', 'id')).map(y => y.contact.contact)
```

Comment:
I think we should be filtering by the name here and not the id.

Reply:
Yep, sounds reasonable. I'll need to change the API call in order to fix this as I need to pull in the contact type records then, or I could add an API call to get a full list of contact_type. Any preference on the two options?

Reply:
In my branch I've just hardcoded for now. Nothing wrong with the approach you've taken, I just think it would be better to search by the proper names.

First option sounds more sensible though.